src/main/java/org/launchcode/water_garden_tour/models/User.java

```diff
@@ -4,31 +4,33 @@
 import javax.persistence.GeneratedValue;
 import javax.persistence.Id;
 import javax.persistence.Table;
+import javax.validation.constraints.NotBlank;
 import javax.validation.constraints.NotNull;
 
 @Entity
-@Table(name = "user")
+//@Table(name = "user")
```

Comment:
if this is not needed we can just remove this

Reply:
another commented out code could be removed if not need. If you are still in a development stage ignore this message

Reply:
Not sure if we need it yet.